scripts/build.mjs

```diff
@@ -0,0 +1,88 @@
+import esbuild from "esbuild";
```

Comment:
This file needs to be linted.
It would be a good idea to add it to the linting script so it gets done automatically

Reply:
Good call! I've done so.